### DISCUSSION TITLE
Make editor instance available in all vue files

### DISCUSSION THREAD

**Discussion by iabhiyaan** [GrapesJS/grapesjs#3598] [Q&A]
How can we make the instance of grapesjs available in all .vue files and .js files ?
E.g

```
 In Main.vue
 mounted() {
  this.editor = grapesjs.init({
      // options
  })
 }
```
What I want is that in other vue component 
ChildComp.vue
```
 mounted() {
  console.log(this.editor.Pages.getAll())
}
```

**Comment by artf** [comment]
Probably you should check Vue documentation on how to achieve that, this has nothing to do with GrapesJS
